src/input-field/index.js

```diff
@@ -159,6 +162,11 @@ export function render({ props, state }, setState) {
     onBlur, onFocus
   };
 
+  // add accept support to <input type=file>s
+  if (type === 'file' && accept) {
```

Comment:
is it really important that `type === 'file'`? I've generally just attached whatever attributes the user passes through. (ie: they can stick "min" and "max" even if they didn't pick a `number` type)